src/pages/Login/components/RedirectHandler.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+
+const RedirectHandler = props => {
```

Comment:
```suggestion
const RedirectHandler = ({ history }) => {
```
이런 식으로 곧바로 구조분해 할당 해서 쓸 수 있습니다